resources/wikia/modules/spec/imageServing.spec.js

```diff
@@ -21,8 +21,8 @@ describe('Image Serving', function () {
 		expect(typeof cutOptions.yOffset2).toBe('number');
 	});
 
-	using('image sizes', [
```

Comment:
I know we already discussed it offline, but maybe someone else has better solution for dataproviders in jasmine.

We introduced dataproviders to help testing same method with different dataset - I expect it as a regular use case. The problem is when test case fails on some set, and you don't know which. In the old version of jasmine dataproviders used a "whole" in jasmine API which allowed overriding failure messages for test case. It was fixed in new version of Jasmine.

Now we end up with manually handling failure message which is not perfect.